README.md

```diff
@@ -194,6 +194,39 @@ In case of authentication failure user is redirected back to login URL with
   a user has been authenticated through provider but your finder has condition
   to not return inactivate user.
 
+### Event Listener
+
+To set up a listener for the `SocialAuth.afterIdentify` event, you can for example
+add this to your `UsersTable::initialize()` method:
+```php
+use Cake\Event\EventManager;
+
+// at the end of the initialize() method
+EventManager::instance()->on('SocialAuth.afterIdentify', [$this, 'updateUser']);
+```
+
+Then create such method in this table class:
+```php
+    /**
+     * @param \Cake\Event\EventInterface $event
+     *
+     * @return \App\Model\Entity\User
+     */
+    public function updateUser(EventInterface $event): User
+    {
+        /** @var \App\Model\Entity\User $user */
+        $user = $event->getData()['user'];
```

Comment:
I believe `$user` instance should be available as 2nd argument of the method too, that would be more convenient for usage.